README.md

```diff
@@ -749,3 +826,16 @@ Run pytest in `path` with `params`.
 Settings:
 
 * `pytest_entry` python entry command (Default: `-m pytest`)
+
+### secrets
+
+#### secrets.setup-env-credentials
+
+Fill specified credentials in your file from k8s.
```

Comment:
Please describe what is `from k8s`. Mention that there should be dot-env file in K8S pod (by specified setting) and this task accept list of var names that should be replaced in `env_file_path` file